src/main/java/org/broadinstitute/hellbender/utils/haplotype/EventMap.java

```diff
@@ -395,6 +396,10 @@ public String toString() {
         return startPosKeySet;
     }
 
+    public Iterator<VariantContext> getSpanningEvents(final int loc) {
```

Comment:
I would call this `getOverlappingEvents` because otherwise it suggests that you're *only* getting spanning deletions.

Reply:
done